### PR TITLE
curvefs/client: Clarify val

### PR DESCRIPTION
**Pull request by shentupenghui**
### What problem does this PR solve?

Issue Number: #2049 

Problem Summary:

### What is changed and how it works?

What's Changed:

1. rename some variables to camel case style
2. correct some spelling mistake

How it Works:

also replace all related files

Side effects(Breaking backward compatibility? Performance regression?):
No

### Check List

- [x] Relevant documentation/comments is changed or added
- [x] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by wuhongsong** [comment]
good job @shentupenghui

File: curvefs/conf/client.conf [hunk 1]
```diff
@@ -109,39 +109,38 @@ volume.bigFileSize=1048576
...
 volume.allocator.type=bitmap
 
 ## for bitmap allocator
 # size of each bit, default is 4MiB
-volume.bitmapallocator.size_per_bit=4194304
+volume.bitmapAllocator.size_per_bit=4194304
 
 # small allocation proportion [0-1]
-volume.bitmapallocator.small_alloc_proportion=0.2
+volume.bitmapAllocator.small_alloc_proportion=0.2
```
Comment (wuhongsong) [inline curvefs/conf/client.conf:120]: please check for changes to the configuration item name to ensure that every where that uses the configuration item has changed.

ditto to other item

Comment (shentupenghui) [reply curvefs/conf/client.conf:120]: updated & checked other item : )

File: curvefs/src/client/common/config.cpp [hunk 2]
```diff
@@ -196,12 +196,12 @@ void InitVolumeOption(Configuration *conf, VolumeOption *volumeOpt) {
...
+        "volume.blockGroup.allocate_once",
         &volumeOpt->allocatorOption.blockGroupOption.allocateOnce);
 
     if (volumeOpt->allocatorOption.type == "bitmap") {
         conf->GetValueFatalIfFail(
-            "volume.bitmapallocator.size_per_bit",
+            "volume.bitmapAllocator.size_per_bit",
             &volumeOpt->allocatorOption.bitmapAllocatorOption.sizePerBit);
         conf->GetValueFatalIfFail(
             "volume.bitmapallocator.small_alloc_proportion",
```
Comment (wuhongsong) [inline curvefs/src/client/common/config.cpp:207]: ditto

Comment (wuhongsong) [inline curvefs/src/client/common/config.cpp:207]: ditto

Comment (wuhongsong) [reply curvefs/src/client/common/config.cpp:207]: conf->GetValueFatalIfFail(
            "volume.bitmapallocator.small_alloc_proportion",

Comment (shentupenghui) [reply curvefs/src/client/common/config.cpp]: im sry for missing one unchanged

Comment (shentupenghui) [reply curvefs/src/client/common/config.cpp]: > conf->GetValueFatalIfFail( "volume.bitmapallocator.small_alloc_proportion",

done

Comment (wuhongsong) [reply curvefs/src/client/common/config.cpp]: good job, lets us wait the ci finished

Comment (shentupenghui) [reply curvefs/src/client/common/config.cpp]: > CI still running, is there some failure occured?

Comment (wuhongsong) [reply curvefs/src/client/common/config.cpp]: > > CI still running, is there some failure occured?

http://59.111.91.248:8080/job/opencurve_multijob/6935/ 

as you can see,  there is running the failover test.

Comment (shentupenghui) [reply curvefs/src/client/common/config.cpp]: the failover test log shows:
```
------------------------------------------------------------------------------
test loop mount umount                                                | FAIL |
mount/umount error
------------------------------------------------------------------------------
```

**Comment by wuhongsong** [comment]
recheck

**Review by wuhongsong — APPROVED** [review]
good job, thank you for your contribution

File: curvefs/src/client/common/config.cpp [hunk 3]
```diff
@@ -196,15 +196,15 @@ void InitVolumeOption(Configuration *conf, VolumeOption *volumeOpt) {
...
 
     conf->GetValueFatalIfFail(
-        "volume.blockgroup.allocate_once",
+        "volume.blockGroup.allocate_once",
         &volumeOpt->allocatorOption.blockGroupOption.allocateOnce);
 
     if (volumeOpt->allocatorOption.type == "bitmap") {
         conf->GetValueFatalIfFail(
-            "volume.bitmapallocator.size_per_bit",
+            "volume.bitmapAllocator.size_per_bit",
```
Comment (Cyber-SiKu) [inline curvefs/src/client/common/config.cpp:204]: Should size_per_bit also use camel case?

File: curvefs/src/client/common/config.cpp [hunk 4]
```diff
@@ -196,15 +196,15 @@ void InitVolumeOption(Configuration *conf, VolumeOption *volumeOpt) {
...
             &volumeOpt->allocatorOption.bitmapAllocatorOption.sizePerBit);
         conf->GetValueFatalIfFail(
-            "volume.bitmapallocator.small_alloc_proportion",
+            "volume.bitmapAllocator.small_alloc_proportion",
```
Comment (Cyber-SiKu) [inline curvefs/src/client/common/config.cpp:207]: ditto

File: curvefs/test/client/common/test_config.cpp [hunk 5]
```diff
@@ -39,11 +39,11 @@ TEST(TestInitVolumeOption, Common) {
     conf.SetUInt64Value("volume.bigFileSize", 1ULL * 1024 * 1024);
     conf.SetUInt64Value("volume.volBlockSize", 4096);
     conf.SetUInt64Value("volume.fsBlockSize", 4096);
-    conf.SetUInt32Value("volume.blockgroup.allocate_once", 4);
+    conf.SetUInt32Value("volume.blockGroup.allocate_once", 4);
```
Comment (Cyber-SiKu) [inline curvefs/test/client/common/test_config.cpp:42]: ditto

File: curvefs/test/client/common/test_config.cpp [hunk 6]
```diff
@@ -39,11 +39,11 @@ TEST(TestInitVolumeOption, Common) {
...
     conf.SetStringValue("volume.allocator.type", "bitmap");
-    conf.SetUInt64Value("volume.bitmapallocator.size_per_bit",
+    conf.SetUInt64Value("volume.bitmapAllocator.size_per_bit",
```
Comment (Cyber-SiKu) [inline curvefs/test/client/common/test_config.cpp:44]: ditto

File: curvefs/test/client/common/test_config.cpp [hunk 7]
```diff
@@ -39,11 +39,11 @@ TEST(TestInitVolumeOption, Common) {
...
                         4ULL * 1024 * 1024);
-    conf.SetDoubleValue("volume.bitmapallocator.small_alloc_proportion", 0.0);
+    conf.SetDoubleValue("volume.bitmapAllocator.small_alloc_proportion", 0.0);
```
Comment (Cyber-SiKu) [inline curvefs/test/client/common/test_config.cpp:46]: ditto

File: curvefs/test/client/common/test_config.cpp [hunk 8]
```diff
@@ -55,11 +55,11 @@ TEST(TestInitVolumeOption, TypeError) {
     conf.SetUInt64Value("volume.bigFileSize", 1ULL * 1024 * 1024);
     conf.SetUInt64Value("volume.volBlockSize", 4096);
     conf.SetUInt64Value("volume.fsBlockSize", 4096);
-    conf.SetUInt32Value("volume.blockgroup.allocate_once", 4);
+    conf.SetUInt32Value("volume.blockGroup.allocate_once", 4);
```
Comment (Cyber-SiKu) [inline curvefs/test/client/common/test_config.cpp:58]: ditto

File: curvefs/test/client/common/test_config.cpp [hunk 9]
```diff
@@ -55,11 +55,11 @@ TEST(TestInitVolumeOption, TypeError) {
...
     conf.SetStringValue("volume.allocator.type", "xxx");
-    conf.SetUInt64Value("volume.bitmapallocator.size_per_bit",
+    conf.SetUInt64Value("volume.bitmapAllocator.size_per_bit",
                         4ULL * 1024 * 1024);
-    conf.SetDoubleValue("volume.bitmapallocator.small_alloc_proportion", 0.0);
+    conf.SetDoubleValue("volume.bitmapAllocator.small_alloc_proportion", 0.0);
```
Comment (Cyber-SiKu) [inline curvefs/test/client/common/test_config.cpp:62]: ditto

File: curvefs/test/client/common/test_config.cpp [hunk 10]
```diff
@@ -55,11 +55,11 @@ TEST(TestInitVolumeOption, TypeError) {
     conf.SetUInt64Value("volume.bigFileSize", 1ULL * 1024 * 1024);
     conf.SetUInt64Value("volume.volBlockSize", 4096);
     conf.SetUInt64Value("volume.fsBlockSize", 4096);
-    conf.SetUInt32Value("volume.blockgroup.allocate_once", 4);
+    conf.SetUInt32Value("volume.blockGroup.allocate_once", 4);
     conf.SetStringValue("volume.allocator.type", "xxx");
-    conf.SetUInt64Value("volume.bitmapallocator.size_per_bit",
+    conf.SetUInt64Value("volume.bitmapAllocator.size_per_bit",
```
Comment (Cyber-SiKu) [inline curvefs/test/client/common/test_config.cpp:60]: ditto

File: curvefs/src/client/common/config.cpp (hunk 3 above)
Comment (shentupenghui) [reply curvefs/src/client/common/config.cpp:204]: yes, for better reading, its good to unify the style. I will do this today.

Comment (shentupenghui) [reply curvefs/src/client/common/config.cpp:204]: yes, for better reading, its good to unify the style. I will do this today.

**Comment by shentupenghui** [comment]
@Cyber-SiKu all done in the latest commit

**Comment by Cyber-SiKu** [comment]
> @Cyber-SiKu all done in the latest commit

LGTM!

**Pull request merged**